Ustring.php

```diff
@@ -520,15 +520,15 @@ function (array $matches) {
                 );
             }
 
-            $string        = static::transcode($string, 'UTF-8', 'ASCII//IGNORE//TRANSLIT');
```

Comment:
Why this change?

Reply:
Because the test fail like this, and I assumed that the behavior expected was to replace by ascii not ignore.

```
There is 1 failure:
~> Hoa\Ustring\Test\Unit\Ustring::case_to_ascii_no_transliterator_no_normalizer_try():
In file /home/pierozi/Project/github/pierozi/Ustring/Test/Unit/Ustring.php on line 953, mageekguy\atoum\asserters\phpString() failed: strings are not equal
-Expected
+Actual
@@ -1 +1 @@
-string(26) "Un ete brulant sur la cote"
+string(22) "Un t brlant sur la cte"
```